### PR TITLE
Check for libnode (additional/instead of libv8)

### PR DESCRIPTION
**Pull request by stesie**
refs #522, picking up the idea to use `libnode`  on alpine

### CODE REVIEW

**Comment by stesie** [comment]
@JoshuaBehrens given you recently worked on the auto-detection, what do you think?

File: config.m4 [hunk 1]
```diff
@@ -3,46 +3,53 @@ PHP_ARG_WITH(v8js, for V8 Javascript Engine,
...
-    if test -r "$i/$PHP_LIBDIR/$SEARCH_FOR" -a -r "$i/include/libplatform/libplatform.h"; then
-      V8_INCLUDE_DIR="$i/include"
-      V8_LIBRARY_DIR="$i/$PHP_LIBDIR"
-      AC_MSG_RESULT(found in $i)
-    fi
+    AC_MSG_CHECKING([for V8 files in default path])
+    ARCH=$(uname -m)
+
+    for i in $SEARCH_PATH ; do
+    echo checking "$i/$PHP_LIBDIR/$SEARCH_FOR" -a -r "$i/include/$libname/v8.h"
```
Comment (JoshuaBehrens) [inline config.m4:29]: There are no other echos here. Is this a remaining debug snippet or is it seen helpful for the overall output of the script?

**Review by JoshuaBehrens — CHANGES_REQUESTED** [review]
Rest of it LGTM as you basically just wrapped a loop around it. With this in mind I assume we need a workflow verifying a `apk add libnode`-ish installation on alpine works out. Right?

File: config.m4 (hunk 1 above)
Comment (stesie) [reply config.m4:29]: yes, that shouldn't actually go there...

**Comment by stesie** [comment]
@JoshuaBehrens 
> With this in mind I assume we need a workflow verifying a apk add libnode-ish installation on alpine works out. Right?

well, I'm unsure if we actually _need_ it. But sure, certainly it is a nice to have. On the other hand I don't want to overact it.

But yes, why not have some jobs next to the matrix one, that builds it on alpine and maybe one on ubuntu.
After all this may also provide guideance to others how the extension can be installed on particular systems

**Comment by JoshuaBehrens** [comment]
@stesie yes, I also thought about test on package manager based builds. They should not take a lot of minutes and ensure others have easy access to this.

**Pull request merged**